core/domain/learner_progress_services.py

```diff
@@ -430,26 +430,6 @@ def _remove_activity_ids_from_playlist(
         learner_playlist_services.save_learner_playlist(learner_playlist)
 
 
-def remove_exp_from_completed_list(user_id, exploration_id):
```

Comment:
Thanks for cleaning i.e, deleting the unwanted codes! :)

Reply:
:)